bundles/org.openhab.binding.teslapowerwall/src/main/resources/OH-INF/addon/addon.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon:addon id="teslapowerwall" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:addon="https://openhab.org/schemas/addon/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/addon/v1.0.0 https://openhab.org/schemas/addon-1.0.0.xsd">
+
+	<type>binding</type>
+	<name>Tesla Powerwall Binding</name>
+	<description>This is the binding for Tesla Powerwall.</description>
+	<connection>local</connection>
+
```

Comment:
Would be nice if you could add suggestion finder xml.

Reply:
I don't believe there is anyway I can detect a powerwall on the LAN.  mdns doesn't detect it...